docs/private-endpoints.rst

```diff
@@ -32,8 +32,10 @@ started. The workflow will look something like this:
 4. We confirm it.
 
 That's it. After this, you will have a secure, private way of connecting to
-your CrateDB Cloud cluster.
+your CrateDB Cloud cluster. Visit the `Cloud Console`_ to configure a private
```

Comment:
This is not strictly speaking true, as the cloud console only asks you to get in touch with us, and there's no automated way of configuring it.